test/test_app_log.c

```diff
@@ -818,7 +823,7 @@ void test_app_log_process_nomem_block (void)
     err_code = app_log_process (&sample);
     TEST_ASSERT (RD_SUCCESS == err_code);
 }
-
+#if     0 // unused functions should not be tested          \/  \/  \/
```

Comment:
Remove

Reply:
I thought we agreed to cancel this previously ?
Is there something I (we) need to do to convince git to stop attempting to update test_app_log.c ??